build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    // Linker workaround for openssl with musl
+    println!("cargo:rustc-link-arg=-lc");
```

Comment:
How about using `.cargo/config` ? 
```toml
[target.x86_64-unknown-linux-musl]
rustflags = ["-C", "link-arg=-lc"]
```
This approach gives us the option to switch the `link-arg` depending on the platform while the build script affects all the platforms.

Reply:
Sounds good. I verified your solution. Let me update the commit.